src/TMC2130Stepper.h

```diff
@@ -4,7 +4,8 @@
 #include <Arduino.h>
 #include <stdint.h>
 
-const uint32_t TMC2130Stepper_version = 0x20200; // v2.2.0
+#define TMC2130STEPPER_VERSION 0x020201 // v2.2.1
```

Comment:
Notice that this bumps the version to 2.2.1, so it's intended for the next release, whenever that would be. I didn't bump the other one to 0.0.5 however.